### PR TITLE
Feature api

### PR DESCRIPTION
**Pull request by ccobasi**
- Create game board
- Implement refresh method
- Implement submit method

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,13 +1,42 @@
...
+const callApi = async () => {
+  const getData = await fetch(url);
+  const resource = await getData.json();
+  createGame(resource);
+  return resource;
+};
+
+callApi();
+
+const addScore = () => {
```
Comment (Davidosky007) [inline src/index.js:25]: * I get this response (status 400) when i try to add a score kindly fix it 🙏 
image

File: src/index.js [hunk 2]
```diff
@@ -1,13 +1,42 @@
...
 
-  // Lodash, imported
-  element.innerHTML = _.join(['Hello', 'webpack'], ' ');
+const createGame = (name) => {
+  leaderboard.innerHTML = '';
+  name.result.map((e, index) => leaderboard.innerHTML += `<li class="list-group-item">${name.result[index].user} ${name.result[index].score}</li>`);
 
-  return element;
-}
+  leaderboard.appendChild(name);
```
Comment (Davidosky007) [inline src/index.js]: * This line is throwing an error as appendChild is expecting a Node but instead got a string(name) please make the necessary adjustments
image

File: README.md [hunk 3]
```diff
@@ -1 +1,59 @@
...
+
+- GitHub: [Chukwuma Obasi](https://github.com/ccobasi)
+- Twitter: [Chukwuma Obasi](https://twitter.com/obasichux)
+- LinkedIn: [Chukwuma Obasi](https://www.linkedin.com/in/chukwuma-obasi/)
+
+## ğ¤ Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/smunozmo/leaderboard/issues).
```
Comment (Davidosky007) [inline README.md]: * I suggest you use the correct issue link for this project as it makes your readme file accurate and professional

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
# Hi @ccobasi 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
* N.A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: src/index.js (hunk 1 above)
Comment (DeliceLydia) [reply src/index.js:25]: Well addressed this!

File: src/index.js (hunk 2 above)
Comment (DeliceLydia) [reply src/index.js]: Well done

File: README.md (hunk 3 above)
Comment (DeliceLydia) [reply README.md]: Well addressed this!

**Review by DeliceLydia — APPROVED** [review]
# Congratulations 🎉 🎉 

## Hi @ccobasi, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard).. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**